docs/micro-rdk/board/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Micro-RDK Board Models"
+linkTitle: "Board"
+weight: 30
+type: "docs"
+description: "Board support in the micro-rdk."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components", "micro-rdk"]
+# SMEs: Nick M., Gautham V.
+---
+
+## Configuration
+
+The micro-RDK currently supports the following models of board:
+
+<!-- prettier-ignore -->
+| Model             | Description              |
+| ----------------- | ------------------------ |
+| [`esp32`](esp32/) | An ESP32 microcontroller |
+
+## API
+
+The [Board API](/components/board/#api) is limited to the following supported client SDK API methods, which operate the same as in the full-featured RDK:
+
+- [`SetPowerMode()`](/components/board/#setpowermode)
+- [`GPIOPin` `Set()`](/components/board/#set)
+- [`GPIOPin` `Get()`](/components/board/#get)
+- [`GPIOPin` `GetPWM()`](/components/board/#getpwm)
+- [`GPIOPin` `SetPWM()`](/components/board/#setpwm)
+- [`GPIOPin` `PWMFreq()`](/components/board/#pwmfreq)
+- [`GPIOPin` `SetPWMFreq()`](/components/board/#setpwmfreq)
+- [`AnalogReader` `Read()`](/components/board/#read)
+- [`DigitalInterrupt` `Value()`](/components/board/#value)
```

Comment:
I see what you're doing here but I think it'd be good to explore more formatting options to make it clear that `GPIOPin`/`AnalogReader`/`DigitalInterrupt` is not part of the method name--maybe one option is take it out of the hyperlink so it's not blue, just to make a bit more visually distinct. Or, break this list out into lists for the methods on each sort of object.